frontend/src/services.ts

```diff
@@ -31,11 +44,45 @@ export const updateTask = async (task: TaskInterface) => {
     }
     // update task with put request
     const response = await axios.put(
-      `${TASK_API_URL}/tasks/${task.id.toString()}`,
+      `${API_URL}/tasks/${task.id.toString()}`,
       task
     );
     return response.data;
   } catch (e) {
     throw new Error("Error in Axios update query to /tasks/<id>");
   }
+
 };
+
+
+export const createTask = async () => {
+  try {
+    // console.log("creating task");
+    // Uses axios to make a put request at "http://localhost:3001/api/tasks"
+    const response = await axios.put(`${API_URL}/tasks`);
+    console.log(response.data.task, " yeahhhh");
+    return response.data.task;
+  } catch (e) {
+    throw new Error("Error in Axios put query to /tasks");
+  }
+};
+
+export const updateMealDelivery = async (mealDelivery: MealDeliveryInterface) => {
+  try {
+    // throw error if no id field is present in mealDelivery object
+    if (!mealDelivery.id) {
+      throw "no id field present in delivery";
```

Comment:
throwing error will crash the app, I think a better approach is just to display an alert saying update failed, and does nothing for now

Reply:
updated!